src/modules/jcms_migrate/migration_assets/subjects.json

```diff
@@ -97,6 +97,13 @@
     "image": "subjects/elife-sciences-biophysics-and-structural-biology-illustration.jpg",
     "alt": "Biophysics and Structural Biology"
   },
+  {
+    "id": "biochemistry-chemical-biology",
+    "name": "Biochemistry and Chemical Biology",
+    "impact_statement": "eLife publishes research spanning chemical biology and enzymology to metabolism and membrane proteins. Decisions are made by <a href=\"https://elifesciences.org/about/people/biochemistry\">journal editors</a> who are active researchers in biochemistry.",
+    "image": "subjects/elife-sciences-biochemistry-illustration.jpg",
+    "alt": "Biochemistry and Chemical Biology"
+  },
```

Comment:
shouldn't this show 2 subjects being created?

Reply:
> shouldn't this show 2 subjects being created?

I'm not sure what you mean, sorry.

Reply:
Have just added `Immunology and Inflammation`

Reply:
That was it :+1: